cranelift-wasm/src/environ/dummy.rs

```diff
@@ -124,6 +125,9 @@ pub struct DummyEnvironment {
 
     /// Instructs to collect debug data during translation.
     debug_info: bool,
+
+    /// Function names.
+    function_names: HashMap<FuncIndex, String>,
```

Comment:
light suggestion: since FuncIndex is an EntityRef, could this be a SecondaryMap instead, for more packing?